gammapy/spectrum/models.py

```diff
@@ -2,24 +2,90 @@
 """Spectral models for Gammapy.
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
+from ..extern.bunch import Bunch
+from . import CountsSpectrum
+import numpy as np
+
 
 __all__ = [
     'SpectralModel',
     'PowerLaw',
     'ExponentialCutoffPowerLaw',
 ]
 
+# Note: Consider to move stuff from _models_old.py here
 
 class SpectralModel(object):
     """Spectral model base class.
     """
+    def __call__(self, energy):
+        kwargs = self.parameters
+        kwargs.update(energy=energy)
+        return self.evaluate(**kwargs)
+
+    def predicted_counts(self, livetime, aeff, edisp):
+        """Get npred 
+        
+        The true and reco energy binning are inferred from the provided IRFs.
+        TODO: make energy binning optional once `~gammapy.irf.EnergyDispersion`
+        has an evaluate method.
 
+        Parameters
+        ----------
+        livetime : `~astropy.units.Quantity`
+            Observation duration
+        aeff : `~gammapy.irf.EffectiveAreaTable`
+            EffectiveArea
+        edisp : `~gammapy.irf.EnergyDispersion`, optional
+            EnergyDispersion
+        """
+        
+        true_energy = aeff.energy.data
+        flux = self.integral(true_energy[:-1], true_energy[1:]) 
+        counts = flux * livetime * aeff.evaluate()
+        counts = counts.decompose()
+        counts = edisp.apply(counts.decompose())
+        return CountsSpectrum(data=counts, energy=edisp.reco_energy)
+        
 
 class PowerLaw(SpectralModel):
     """Spectral power-law model.
+    
+    .. math:: 
+
+        F(E) = F_0 \\cdot \\left( \\frac{E}{E_0} \\right)^{\Gamma}
```

Comment:
Please use "raw" literal docstrings, i.e. add an `r"the docstring"` prefix, and then you can remove the double backslashes from the LaTeX formula, i.e. have a normal LaTeX string.